### PR TITLE
Update the README

### PR DESCRIPTION
**Pull request by gemmaleigh**
- Add details about the Sass dependency on the GOV.UK front end toolkit
- Show how the Sass files within /public/sass/elements are imported
- Explain which files are used only to style the page furniture
- Add an example for running Wraith
- Request that issues are raised with this repository

cc. @fofr, @robinwhittleton.

### CODE REVIEW

**Comment by timpaul** [comment]
Looks good to me, but would be good to run by someone from outside GDS to check it makes sense to them.

**Comment by cjforms** [comment]
Is there a way that I can just read this document as it will look?

**Comment by gemmaleigh** [comment]
Hi @cjforms, here you go: https://github.com/alphagov/govuk_elements/blob/update-readme/README.md

**Comment by robinwhittleton** [comment]
Can we make the commands in ‘running this site locally’ the box-out code style?

**Comment by gemmaleigh** [comment]
HI @robinwhittleton, that's been updated: https://github.com/alphagov/govuk_elements/blob/update-readme/README.md

File: README.md [hunk 1]
```diff
@@ -1,80 +1,130 @@
...
+Go to [localhost:3000](http://localhost:3000) in your browser.
+
+### Running wraith to compare changes
+
+GOV.UK elements uses Wraith so that regressions can be easily spotted.
+Runs against all the main elements pages and some of the examples.
+
+#### Usage
+
+On master run:
```
Comment (fofr) [inline README.md]: This needs to be run from the Wraith directory and some dependencies need to be installed on the local machine first.

File: README.md [hunk 2]
```diff
@@ -1,80 +1,130 @@
...
+
+    wraith history config.yaml
+
+
+On local branch run
+
+    wraith latest config.yaml
+
+
+##  How to we want people to contribute and improve elements.
```
Comment (fofr) [inline README.md]: There's a typo here. There's a Github pattern that puts contribution guidance in a file named `CONTRIBUTING.md`, eg https://github.com/alphagov/govuk_frontend_toolkit/blob/master/CONTRIBUTING.md

**Comment by gemmaleigh** [comment]
Hi @fofr, @robinwhittleton - is this OK to merge?

**Comment by fofr** [comment]
@gemmaleigh The local dependencies for Wraith are:
```
gem install wraith
brew install phantomjs
brew install imagemagick
```

**Comment by robinwhittleton** [comment]
👍

**Pull request merged**